### ISSUE TITLE
why sigma=9?

### ISSUE THREAD

**Issue by bashhwu** [pawni/BayesByHypernet#3]
Hi Pawni,

In the error loss function, sigma=9.0
Why did you choose such a value?

if we choose another value, we should retune the hyperparameters, right?

**Comment by pawni** [comment]
I assume you're talking about the scale for the data generation? That is just reused from Hernandez-Lobato & Adams (2015).

**Comment by bashhwu** [comment]
Actually, I am talking about the scale used in the log-likelihood term in the loss function.

**Comment by pawni** [comment]
That is because it is the true noise value from the generation. Normally you would want you're method to predict that sigma but we just fix it to the true value similar to some prior works.
